cpu/saml21/Makefile.include

```diff
@@ -3,5 +3,17 @@ export CPU_FAM  = saml21
 
 USEMODULE += pm_layered
 
+ifneq (,$(filter saml21j18b saml21j18a,$(CPU_MODEL)))
```

Comment:
Can you also add SAMR30 here please ? We might find a better way for this later to avoid adding new MCU.

Reply:
Ok.

Reply:
Maybe
```mk
ifneq (,$(filter saml21,$(CPU_FAM)))
```
?

Reply:
They don't all have the same amount.
SAML21x18 & SAML21x17 have 8k, SAML21x16 has 4k and SAML21E15 comes with 2k LP-SRAM.